lang/rs/apis/cne/src/port.rs

```diff
@@ -85,6 +114,20 @@ impl Port {
         Ok(pkts_read)
     }
 
+    /// Prepares burst of [packets](Packet) to be sent from the port.
+    /// This function allocates underlying buffers for the packets in the slice.
```

Comment:
Can you add a blank line here to split up the "short description" from the "detailed description"?

Reply:
Done